### PR TITLE
Adds ipycytoscape ipygany nbs

### PR DESCRIPTION
**Pull request by marimeireles**

### CODE REVIEW

File: notebooks/07.05-ipycytoscape.ipynb [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
...By/ZCxQcjpMEnix2sYbjuHrQ3qpWNiepowY2+2huqqZ3VlfSzkN1PDlup2H98uiqqf3oQwuG04CyHMrgAAsi3UMApOjppbXq4BH9MumyUWytcB6BFcM6DkSxlaO1Q2Cxe/usIgkGdx7cZPe5Edh6qJlWsx/V/Am5HBAy7dwH22TvoWNNdNf/vEVVtc0c9bWNfD4m6zyypiXzgMxmbvMm9aPffPkyHkClT7LJLZNiCAKCQAwhAFPjF1eeUvOPu2aHKllsDAEkVYkoArDoOHrKS4s2c1A5y20PFh7uZi/99G8LydXs8ZPwYPN2TczNlgBRD9aqD+lfTB+/bRrNHJWr/OJhci/iXARiQoMO+JMS2ugvL6yl/UeqeRLcrhq4kV2VdYTXn1/eSVNHltDX7ptIl40vde5di2DJ09mcuLLqFG0/cJKea3RTcU4SFedn0PihhXT1pf2ptEib7pgigdAvYTOe4/Xa33wW51ZEpHa7BvWKIJRyqR4i4GGfKjzWdozgfraqDSjNphV/upVaOIBDk9tHSBMHTU4GB3PIyUyhZPE3Pxt0sl0QEAQEgR4j0MqZP6B5lDlIj6GUE4QAAbjB9WNT83uvKOb5QDsdZTfQxVsbaOXWQ0zO2Uwdanee55xuzo5Lg9UEMsKc6bBRefwUp3nLpmF9M2zrBhgCCOPmFDFB0I/VumnB515k8xHtg2FWmvRdtBoxv7Vyro3l245xlOU36DdfmEV3zx8iQUPO0tTdTLSXbT5KP/rbetpcUaOOCkSd1j/KZJJxz/xh9PFbRtHQvnkcwdLHq4KNOhgc9yRXjMul8t5p0lGcBWPZ3H0E4Ku10TLbyuDUf04TpETBS0QQEAQEAUEgcgi4OF+1in4tUWkjB7pc6bwIgIfDV30IB5QbwHnVH395tf7N6TbNfFxCRzAZ1zQ9WAlpyLyH07FVHK6l4f3Ob+l63gLKAVFHwPEzxsMnXTTzky9QE5uFBBNI1YStSFLKit80ev6MXIJffeh9ZW764etGRP0m2K0AbZwi6nO/WEovrzjIQfeCou75vSG0eY3b46VHX9tJLy0/QF+4axLlFhRxrkY+nnG/dgqnleAciyKCQCgQaOPAJ3ieMRClJZ+5chyKa8g5BAFBQBAQBGILARdn+IFb0YTBEpsotu5s7NQG6ddq6lyWhlzXy+9vzuQ82Lwde9k3Wc+HrDl5
```
Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:328]: I get this error when I execute this cell:

```python-traceback

---------------------------------------------------------------------------
NameError                                 Traceback (most recent call last)
<ipython-input-18-ce1f2016e705> in <module>
----> 1 customInheritedGraph.graph.nodes

NameError: name 'customInheritedGraph' is not defined
```

File: notebooks/07.05-ipycytoscape.ipynb [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+   "metadata": {},
+   "source": [
+    "### ipycytoscape walkthrough\n",
+    "\n",
+    "  - cytoscape json\n",
+    "    - layouts\n",
+    "  - networkx\n",
+    "    - directed and undirected graphs\n",
+    "    - creating custom elements\n",
+    "  - pandas\n",
```
Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:38]: Consider using the pandas example to be the first or second example. This particular audience is likely to have lots of pandas users.

File: notebooks/07.05-ipycytoscape.ipynb (hunk 1 above)
Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:161]: Maybe encourage them to try a couple of other layouts here? Might not be time for that, though...

Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:285]:
```suggestion
    "#### **Creating custom elements**\n",
```

Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:286]: Maybe add a link to the API here?

Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:498]:
```suggestion
    "More information about cytoscape selectors is available [here](https://js.cytoscape.org/#selectors)."
```

Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:516]:
```suggestion
    "More examples of how to set labels and tooltips you can find [here](https://github.com/cytoscape/ipycytoscape/blob/master/examples/Text%20on%20node.ipynb) and [here](https://github.com/cytoscape/ipycytoscape/blob/master/examples/Tooltips%20example.ipynb). Or online via [binder](https://mybinder.org/v2/gh/QuantStack/ipycytoscape/stable?filepath=examples)."
```

Comment (mwcraig) [reply notebooks/07.05-ipycytoscape.ipynb:516]: Github seems to be really confused about this suggestion. Basically it is a few minor wording changes:

`More examples on how to set and labels` → `More examples of how to set labels and tooltips`

and

`nad` → `and`

Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:545]:
```suggestion
    "### **Interactivity and interoperability with ipywidgets**\n",
```

File: requirements.txt [hunk 3]
```diff
@@ -13,9 +13,11 @@ jupyterlab==3
 numpy
 pandas
 pythreejs==2.3.0
+pyvista=0.31.3
```
Comment (mwcraig) [inline requirements.txt:16]:
```suggestion
pyvista==0.31.3
```

File: requirements.txt [hunk 4]
```diff
@@ -13,9 +13,11 @@ jupyterlab==3
...
 requests
 scikit-image
 scipy
 sidecar==0.5.1
 voila==0.2.10
 voila-vuetify==0.5.2
+vtk=8.2.0
```
Comment (mwcraig) [inline requirements.txt:23]:
```suggestion
vtk==8.2.0
```

File: notebooks/07.06-ipygany.ipynb [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "940761d1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Preparing example's data\n",
```
Comment (mwcraig) [inline notebooks/07.06-ipygany.ipynb:45]: Maybe add a brief explanation before this that you are loading data of elevations on the Earth.

Comment (mwcraig) [reply notebooks/07.06-ipygany.ipynb:45]: Also say that you are demonstrating warping.

File: notebooks/07.06-ipygany.ipynb [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6d946612",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipygany import TetraMesh,Threshold"
```
Comment (mwcraig) [inline notebooks/07.06-ipygany.ipynb:165]: Explain with some text that you are demonstrating thresholding here.

**Review by mwcraig — COMMENTED** [review]
Thanks @marimeireles these both look good! I have some inline comments, mostly minor.

Can you please add the two new requirements to `environment.yml`?

File: notebooks/07.05-ipycytoscape.ipynb (hunk 1 above)
Comment (mwcraig) [reply notebooks/07.05-ipycytoscape.ipynb:285]: Hmmm, the change here was just `creating` → `Creating`

**Comment by martinRenou** [comment]
I will push the necessary changes so that vtk works on binder :)

**Comment by marimeireles** [comment]
Thanks for the review @mwcraig =)
and thanks Martin!

File: notebooks/07.05-ipycytoscape.ipynb (hunk 1 above)
Comment (marimeireles) [reply notebooks/07.05-ipycytoscape.ipynb:161]: The problem with this is that we need to load different kinds of datas to actually be able to show the different kinds of layout. I'd say it's not worth it.
But I could remove the image, if it's a problem.

Comment (marimeireles) [reply notebooks/07.05-ipycytoscape.ipynb:286]: You mean instead of listing the layouts?

Comment (marimeireles) [reply notebooks/07.05-ipycytoscape.ipynb:286]: Or just changing the position of `    "More information about cytoscape selectors is available [here](https://js.cytoscape.org/#selectors)."`

File: notebooks/07.05-ipycytoscape.ipynb [hunk 7]
```diff
@@ -539,7 +542,7 @@
   },
   {
    "cell_type": "markdown",
-   "id": "aacd736a",
+   "id": "6805e88d",
    "metadata": {},
    "source": [
     "### **interactvicity and interoperability with ipywidgets**\n",
```
Comment (mwcraig) [inline notebooks/07.05-ipycytoscape.ipynb:548]: Just a minor change....

```suggestion
    "### **Interactivity and interoperability with ipywidgets**\n",
```

**Comment by mwcraig** [comment]
Testing out the new workflow to generate binder links on PRs: /binder

**Comment by mwcraig** [comment]
Close/reopen to try getting binder link....

**Comment by mwcraig** [comment]
@marimeireles  -- is this ready to merge?

**Comment by mwcraig** [comment]
I'm going to merge so I can start some re-arranging 😃

**Comment by marimeireles** [comment]
No problem. Just finished to fix the small changes here on both notebooks and will push soon.
Sorry for taking so long :see_no_evil: I ended up taking yesterday and today off cause I had to attend to some personal things.
Thank you for all your work @mwcraig! =)

**Comment by mwcraig** [comment]
No worries -- you will need to refresh your master because Itay and I just did a big rearrangement.

**Pull request merged**